src/content/guides/pay-payment-app-integration.mdx

```diff
@@ -0,0 +1,139 @@
+  order: 1
+---
+
+## Overview
+
+This guide is for a [Third-party Asset Provider](/guides/integrating-third-party-asset) to enable paying with their
+digital assets via Centrapay.
+
+## Authenticating API calls on behalf of users
+
+Third-party applications can make API calls on behalf of users with Centrapay
+API keys. All API requests to Centrapay must be made from the integrator’s backend.
+Centrapay API keys must also be managed on the backend and stored as secrets to
+minimize the risk of API keys being compromised.
+
+Seperate API keys will be used for test and live payments.
+
+> Contact [integrations@centrapay.com](mailto:integrations@centrapay.com) to be issued your API keys.
+
+## Payment Flow Implementation
+
+The payment flow below describes how a Third-party Asset Provider should integrate
+with the [Dynamic Merchant QR Code flow](/guides/payment-flows#dynamic-merchant-qr-code) to enable digital payments within their
+application. Below is an overview of the process, followed by a more in-depth
+look at the integrator’s responsibilities.
+
+> Note this is assuming the best case scenario, and not accounting for edge cases.<br/><br/>
+> You should handle all documented and unknown errors how you see fit and validate if a payment is payable before making requests to avoid a bad user experience.
+
+```mermaid
+sequenceDiagram
+  autonumber
+
+	actor user
+	participant t as Terminal/POS
+	participant a as App
+	participant b as Integrator's Backend
+	participant c as Centrapay
+
+	user->>t: Select Centrapy
+	t->>c: Create Payment Request
+	note over t: Display QR code
+	a->>t: Scan QR code
+	note over a: Parse Centrapay QR code
+	a->>b: Request payment details
+	b->>c: GET /payment-request/{id}
+	a->>user: Display Payment Request Details
+  user->>a: Approves payment
+	a->>b: Initiate payment
+	b->>c: POST /payment-request/{id}
+	a->>user: Display successful payment
+```
+
+### User selects Centrapay as their payment method
+
+A payment terminal will provide an option for Centrapay that the user must press.
+A Payment Request will then be created by the merchant and the url field of the
+newly created Payment Request will be presented as a QR code.
+
+![Concept image of terminal displaying Centrapay QR code](/terminals/displaying-centrapay-qr-code.png)
+
+### User scans the Centrapay QR code via integrated app
+
+The user will then scan the QR code from inside your application. The valid Centrapay QR Code
+will have the format `https://app.centrapay.com/pay/{paymentRequestId}`. Validate
+the url by asserting it matches the format and ignore any malformed QR codes.
+
+### Get the Payment Request’s details
+
+Once your application has validated the url, use the id of the Payment Request to
+call the [Get Payment Request](https://docs.centrapay.com/api/payment-requests#get-a-payment-request)
+API from your backend. The returned [PaymentRequest model](https://docs.centrapay.com/api/payment-requests#payment-request)
```

Comment:
```suggestion
API. The returned [PaymentRequest model](https://docs.centrapay.com/api/payment-requests#payment-request)
```

Reply:
This is now call the 'call the [Get Payment Request](https://docs.centrapay.com/api/payment-requests#get-a-payment-request)
API from your server.' Think it is important to still mention the app should not be calling our API's directly.